pos_backend_partner/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+POS Backend Partner
+===================
+
+Choose a POS customer from the backend.
+
+
+- Replace customer selection screen in POS by a backend view.
+- Perf improvement : Full customer list is not downloaded anymore by the POS at startup.
```

Comment:
This should be placed in the known limitations

Reply:
It's not a limitation:  it's the main feature ! :)

Reply:
hehe OK :)